### ISSUE TITLE
Skin do not apply on boot

### ISSUE THREAD

**Issue by bzz11g** [theawless/EventLock#21]
Hi, i use your app on OnePlus 5 stock Oreo 8.1, but after each reboot i need to go in EventLock app and press preview button to update widget on lock screen and change design from default to my skin
Xposed log https://drive.google.com/file/d/12DCDCOMGTgui3aUgsmwmva-2U8GQaBf_/view?usp=drivesdk
please do someting)

**Comment by theawless** [comment]
Interesting! It's definitely a bug, but I never noticed it :o

**Comment by theawless** [comment]
Does the preview button update it alone? Or do you need to re-edit the settings and then press the preview button?

**Comment by bzz11g** [comment]
Yes, i just need open app, and press refresh button, if you need i can send
you some logs

пн, 28 мая 2018 г., 21:26 Abhinav Singh <notifications@github.com>:

**Comment by theawless** [comment]
https://play.google.com/store/apps/details?id=fr.simon.marquis.preferencesmanager&hl=en_US

If possible can you install this or similar application, and check whether the preferences of EventLock are same when you boot up and when you press refresh button?

**Comment by bzz11g** [comment]
I didn't see differences

пн, 28 мая 2018 г., 21:42 Abhinav Singh <notifications@github.com>:

**Comment by bzz11g** [comment]
https://imgur.com/a/wJvfN5b

**Comment by theawless** [comment]
Do the preferences related to title/date/calenders work? I see you've added a string for "after", does it show properly just after the boot up?

**Comment by bzz11g** [comment]
yes, all is good with this https://imgur.com/a/DmfgsFX


--add
and immediately after boot too

**Comment by theawless** [comment]
Can you check the permissions of the preferences xml just after the boot and just after you've pressed refresh button?

You can find the preferences at `/data/data/[package name]/shared_prefs/xxx.xml`

**Comment by bzz11g** [comment]
all looks the same
https://imgur.com/a/UsQjVRU

**Comment by bzz11g** [comment]
may be you need dmsg log or something else?

**Comment by theawless** [comment]
Sure. Maybe that'll help :)

**Comment by bzz11g** [comment]
you can grab my logs here https://drive.google.com/folderview?id=1Au6m_7K39_Dv9Q75yZ5mPtWVeq8cTk1X

**Comment by theawless** [comment]
I'm not able to figure it out and logs don't tell me anything extraordinary. :( Pressing the refresh button and creating the view after bootup does the same thing. It fetches the style from the same preferences xml (which has world-readable permission). Maybe it's related to your xposed version?

**Comment by bzz11g** [comment]
maybe, I use latest version of xposed for magisk 90.2-beta3 systemless, maybe this is bug only on Oreo, maybe OxygenOS adding bug's in work of your modul

**Comment by arboeh** [comment]
I have the same issue with my OnePlus 5 (A5000) and Lineage 15.1.
I'm using the same Xposed version...

**Comment by bzz11g** [comment]
i am on AOSPExtended 8.1 now, version of app 3.4 with new icon, and bug still there
